frontend/src/main/pages/ProfilePage.js

```diff
@@ -3,18 +3,76 @@ import { Row, Col } from "react-bootstrap";
 import RoleBadge from "main/components/Profile/RoleBadge";
 import { useCurrentUser } from "main/utils/currentUser";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+
 
 const ProfilePage = () => {
+    
 
     const { data: currentUser } = useCurrentUser();
+    const [show, setShow] = useState(false);
+    //Stryker disable all
+    const [phoneNumber, setPhoneNumber] = useState("");
+    const [updatedPhoneNumber, setUpdatedPhoneNumber] = useState("");
+    //striker enable all
```

Comment:
```suggestion
    // Stryker restore all
```
This is the proper version of the comment that restores Stryker testing.